js_modules/dagit/packages/core/src/assets/AssetMaterializationTable.tsx

```diff
@@ -57,48 +57,58 @@ export const AssetMaterializationTable: React.FC<{
   );
 };
 
+const NoneSpan = <span style={{color: ColorsWIP.Gray400}}>None</span>;
```

Comment:
The callsites for this are slightly confusing since they're not JSX, mind just making this a component and using `<NoneSpan />`?

Reply:
Ahh yeah I'll switch it 🙌 For dataviz components I've been doing a lot of this style of re-use because it cuts down on the memory thrashing for totally static elements, but I don't think it matters with less than like 1000 of them on-screen and is definitely weird.